scripts/dev/dockerfile_generator.py

```diff
@@ -9,6 +11,43 @@
 GOLANG_TAG = "1.14"
 
 
+def _shared_agent_params() -> DockerParameters:
+    with open("release.json", "r") as f:
+        release = json.loads(f.read())
+
+    return {
+        "template_path": "scripts/dev/templates/agent",
```

Comment:
[q] why is this is shared if we always replace that?

Reply:
both agents have the same template path, but currently all the other images are just in "scripts/dev/templates"

Reply:
ah just seeing now you mean we replace it in both but with the same value - this is a mistake! Will correct it.